monitoring/README.md

```diff
@@ -49,26 +89,51 @@ wds1-ks-transport-controller   64m
 b) ITS space:
 
 ```bash
-$ ./configure-metrics-its.sh
+./configure-monitoring-its.sh 
 ```
 
-If you have more than one ITS, you can run the above script with the `--space-name` flag. For example: `./configure-metrics-its.sh --space-name its2`
+If you have more than one ITS, you can run the above script with the `--space-name` flag. For example: `./configure-monitoring-its.sh --space-name its2`
 
 Optionally, check the prometheus service monitor objects for the ITS space:
 
 ```bash
-$ kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep its1*
+kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack" | grep its1*
+```
+
+Output:
+```bash
 its1-apiserver                 64m
 its1-status-addon-controller   64m
 ```
 
+c) Workload execution cluster (WEC):
+
+Run the following command for every WEC: `$ ./configure-monitoring-wec.sh --wec-cluster-context <wec-ctx-name>`.
+
+For example:
+```bash
+./configure-monitoring-wec.sh --wec-cluster-context cluster1
+```
+
+Optionally, check the prometheus service monitor object:
```

Comment:
I think your intent is that this "Optionally..." is inside the body of the above loop over "every WEC", but textually that is not clear; this "Optionally..." appears _after_ the loop and at the same level of indentation as the loop.

Reply:
I added some new text to make it clear that the "optionally" statement is intent for every WEC. Also added the `--context` flag to avoid any ambiguity.